providers/onshape/onshape_provider/onshape_actions.py

```diff
@@ -1,7 +1,11 @@
+from pydoc import cli
+from typing import Optional
+from xmlrpc.client import boolean
```

Comment:
Are `pydoc.cli` and `xmlrpc.client.boolean` on lines 1 and 3 necessary imports? Please remove them if they are not. Thank you.